publication-rest/src/main/java/no/unit/nva/publication/update/PublishingRequestResolver.java

```diff
@@ -115,33 +142,91 @@ private TicketEntry persistPublishingRequest(
                    : publishingRequest.persistNewTicket(ticketService);
     }
 
-    private void persistPendingPublishingRequest(Publication oldImage, Publication newImage) {
+    private void persistPendingPublishingRequest(Publication oldImage, Publication newImage)
+        throws ApiGatewayException {
         var files = getNewPendingFiles(oldImage, newImage).collect(Collectors.toSet());
         var resource = Resource.fromPublication(newImage);
         var workflow = lookUp(customer.getPublicationWorkflow());
+
+        if (resource.isDegree()) {
+            var publisher = getPublisher(resource);
+            if (publisher.isPresent()) {
+                var channelClaim = getChannelClaim(publisher.get());
+                if (channelClaim.isPresent() && !isClaimedByUserOrganization(channelClaim.get(), userInstance)) {
+                    var organizationId = channelClaim.get().claimedBy().organizationId();
+                    FilesApprovalThesis.create(resource, userInstance, organizationId, workflow)
+                        .withFilesForApproval(files)
+                        .persistNewTicket(ticketService);
+                    return;
+                }
+            }
+            FilesApprovalThesis.createForUserInstitution(resource, userInstance, workflow)
+                .withFilesForApproval(files)
+                .persistNewTicket(ticketService);
+        }
         var publishingRequest = PublishingRequestCase
                                     .createWithFilesForApproval(resource, userInstance, workflow, files);
 
         attempt(() -> persistPublishingRequest(newImage, publishingRequest));
     }
 
+    private boolean isClaimedByUserOrganization(ChannelClaimDto channelClaim, UserInstance userInstance) {
+        return Optional.ofNullable(channelClaim)
+                   .map(ChannelClaimDto::claimedBy)
+                   .map(CustomerSummaryDto::id)
+                   .map(id -> userInstance.getCustomerId().equals(id))
```

Comment:
Can customerId from user instance be null?

Reply:
Do not think so. We consume `UserInstance` from Authorized user which performs `/update` on publication

Reply:
I also think "think", which indicates we should always use Optional to indicate things may be null in EVERY case very, very strictly.